doc/kernel/format_specifications.rdoc

```diff
@@ -142,77 +156,40 @@ of the formatted field:
 
 ==== Specifiers +b+ and +B+
 
-Convert +argument+ as a binary integer:
-
 The two specifiers +b+ and +B+ behave identically
-except when +0b+ or +0B+ is prefixed.
+except when flag +#+ is used.
 
-Flags (all are allowed):
+Convert +argument+ as a binary integer:
 
-- No flag:
+  sprintf('%b', 1)  # => "1"
+  sprintf('%b', 4)  # => "100"
+  # Prefix '..' for negative value.
+  sprintf('%b', -4) # => "..100"
 
-    sprintf('%b', 1)  # => "1"
-    sprintf('%b', 4)  # => "100"
-    # Prefix '..' for negative value.
-    sprintf('%b', -4) # => "..100"
+Type-specific modifier:
 
-- Flag <tt>#</tt>: Prefix +0b+ or +0B+:
+- '#' flag' (use alternate format):
 
     sprintf('%#b', 1)  # => "0b1"
     sprintf('%#B', 1)  # => "Bb1"
```

Comment:
This example is wrong, should be `sprintf('%#B', 1)  # => "0B1"`

Reply:
Fixed.